walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,13 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
+    while (name.charAt(0) == ' ') {
+        name = name.substring(1);
+        System.out.println("name");
+    }
+    while (name.charAt(name.length() - 1) == ' ') {
+        name = name.substring(0, name.length() - 1);
+    }
```

Comment:
looks like you're trying to remove whitespace
.trim() should take care of it
(Also remove the debug statement System.out.println)